h2d/Layers.hx

```diff
@@ -1,5 +1,10 @@
 package h2d;
 
+/**
+ * h2d.Layers allows to hierarchically organize objects on different layers and supports Y-sorting.
+ * Note: 
+ * 	 If you need a class that supports different scale modes and alignments see h2d.Scene.

```

Comment:
I'm not sure I understand this note in the context of h2d.Layers.

Reply:
Removed the note.
You're right, I was trying to teach the user that there is a `Scene` class in case he/she needs, but by the time the user finds out about `h2d.Layers` he/she should know the existance of `h2d.Scene` (because of `s2d:h2d.Scene`).